doc/source/tutorial/stats/resampling_tutorial_1.rst

```diff
@@ -0,0 +1,410 @@
+.. _monte-carlo:
+
+Monte Carlo Tests
+-----------------
+
+.. currentmodule:: scipy.stats.resampling
+
+To motivate resampling methods, suppose we wish to infer from
+measurements whether the weights of adult human males in a medical study
+are normally distributed
+[`1 <https://www.jstor.org/stable/2333709https://www.jstor.org/stable/2333709>`__].
+This may be prerequisite to performing other statistical tests, many of
+which are developed under the assumption that samples are drawn from a
+normally distributed population (although some tests are quite robust to
+deviations from this assumption, as we will see later). The weights are
+recorded in the array ``x`` below.
+
+    >>> import numpy as np
+    >>> x = np.array([148, 154, 158, 160, 161, 162, 166, 170, 182, 195, 236])  # weights (lbs)
+
+One way of testing for departures for normality, chosen based on its
```

Comment:
"departures from normality"?

Reply:
Oops, it's a [pretty common phrase](https://www.google.com/search?q=%22departures+from+normality%22&rlz=1C1ONGR_enUS930US930&oq=%22departures+from+normality%22&aqs=chrome..69i57j0i22i30j0i390l4.879j0j7&sourceid=chrome&ie=UTF-8) meaning "ways in the which the data are not normally distributed". If that seems like jargon, I can use a more descriptive phrase.

Reply:
>  If that seems like jargon, I can use a more descriptive phrase.

No, I think it would read fine, but you have written "departures for normality" which didn't seem to flow off the tongue as well as well "departures from normality"

Reply:
Oh, definitely a typo. Thanks!